app/assets/stylesheets/common/_navigation.scss

```diff
@@ -439,7 +423,7 @@ li#dropdown {
   }
 
   @include breakpoint(mobile) {
-    @include border-radius(0);
+    border-radius: 0;
```

Comment:
Properties should be ordered border, border-radius, display, min-width, padding, text-indent, width